drools-wb-services/drools-wb-verifier/drools-wb-verifier-api/src/main/java/org/drools/workbench/services/verifier/api/client/reporting/Issue.java

```diff
@@ -64,4 +64,36 @@ public String getDebugMessage() {
     public void setDebugMessage( final String debugMessage ) {
         this.debugMessage = debugMessage;
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+
+        Issue issue = (Issue) o;
+
+        if (severity != issue.severity) {
+            return false;
+        }
+        if (rowNumbers != null ? !rowNumbers.equals(issue.rowNumbers) : issue.rowNumbers != null) {
+            return false;
+        }
+        if (checkType != issue.checkType) {
+            return false;
+        }
+        return debugMessage != null ? debugMessage.equals(issue.debugMessage) : issue.debugMessage == null;
+    }
+
+    @Override
+    public int hashCode() {
+        int result = severity != null ? severity.hashCode() : 0;
+        result = 31 * result + (rowNumbers != null ? rowNumbers.hashCode() : 0);
+        result = 31 * result + (checkType != null ? checkType.hashCode() : 0);
+        result = 31 * result + (debugMessage != null ? debugMessage.hashCode() : 0);
+        return result;
```

Comment:
GWT Client side and hashCode tend to cause trouble, the workaround is to use ~~

Like this:
https://github.com/kiegroup/drools-wb/blob/1fabbdea01b75ac325e97df72d58d87a65d3955d/drools-wb-services/drools-wb-verifier/drools-wb-verifier-api/src/main/java/org/drools/workbench/services/verifier/api/client/index/FieldBase.java#L100-L105